ReactiveCocoa/Swift/DynamicProperty.swift

```diff
@@ -52,24 +54,12 @@ import enum Result.NoError
 	public init(object: NSObject?, keyPath: String) {
```

Comment:
I think I'd rather keep the initializers here.

We should consider trying to remove the `Optional` here in a separate PR though.